modular_skyrat/modules/GAGS/greyscale_configs.dm

```diff
@@ -42,6 +42,18 @@
 	name = "Beret With Fancy Badge Worn - Vox"
 	icon_file = 'modular_skyrat/modules/GAGS/icons/beret_vox.dmi'
 
+/datum/greyscale_config/beret_badge_fancy/worn/teshari
```

Comment:
just a note, try to sort this by type, not by species; basically how the vox does it:
item
item/worn
item/worn/vox
(then you'd have item/worn/tesh)